Readme.md

```diff
@@ -4,21 +4,26 @@ Thanks to your [Part 107 Commercial Drone Pilot's license](https://www.faa.gov/u
 
 ## Requirements
 
-We have provided a starter web application that has just about everything you need to accomplish the task. The application setup instructions are below. Once you have the app up and running, you will see the polygon of controlled "no fly" airspace on the map. Using the sketch tool you can draw shapes of your various flight areas on the map. After a sketch is complete, your work should display which portion - if any - of the sketch is in controlled airspace (i.e. any area that intersects the controlled airspace polygon).
+We have provided a starter web application that has just about everything you need to accomplish the task. The application setup instructions are below. Once you have the app up and running, you will see the polygon of controlled "no fly" airspace on the map. Using the sketch tool you can draw shapes of your various flight areas on the map. This challenge should not take more than a few hours.
 
-- Search the code for "HINT" - these will help you along
+- Search the code for "HINT" - these will help you along. The MapStore has links to all the documentation you will need.
 - Display a message in the `Info` component indicating whether this flight will be denied (**it intersects**) or approved (**it does not intersect**)
 - Display the area (in sq meters or sq kilometers) of the intersection, if any
 - It'd also be great to see the intersection shape highlighted with a different color so that it is easy to visualize
+- Write some basic CSS to make the `Info` component a little more visually pleasing. There is no one "right" way to do this - get creative!
+- Write a couple simple media queries to adjust the width of the `Info` component as the browser width is narrowed, such that the sketch tools do not overlap
 - When you are finished, run `yarn lint` and fix any linting errors that may have been introduced by your code
```

Comment:
Feel like we shouldn't hint that, maybe we just see if they do this

Reply:
I'm just talking about the `lint` part for this, but yeah most of the hints make sense

Reply:
also I don't think I could get `yarn lint` to execute in my dev environment the day I did the challenge.  I tried several versions of node to no avail, and got tired of messing with it.